src/main/java/sg/ncl/MainController.java

```diff
@@ -2341,7 +2356,7 @@ public String validateExperiment(
 //			catch (Exception e) {
 //				redirectAttributes.addFlashAttribute(MESSAGE,
 //						"You failed to upload " + networkFile.getOriginalFilename() + " => " + e.getMessage());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)